dash/development/base_component.py

```diff
@@ -94,6 +94,14 @@ def __init__(self, **kwargs):
                         ", ".join(sorted(self._prop_names))
                     )
                 )
+
+            if k != "children" and isinstance(v, Component):
+                raise TypeError(
+                    "Component detected as a prop other than `children`\n" +
+                    "Did you forget to wrap multiple `children` in an array?" +
```

Comment:
Either a `\n` or whitespace after the end of the sentence

Reply:
good call -> https://github.com/plotly/dash/pull/1026/commits/4d96e0dbf3c96ec19683c4e68c9e350302cb6da8